packages/docs/docs/spline.md

```diff
@@ -201,15 +201,7 @@ RTF = React Three Fiber
 9. Add the following imports at the **top** of **Torus.tsx**, below the useSpline import.
 
    ```tsx
-   import React, { useEffect } from "react";
-   import { useThree } from "@react-three/fiber";
-   import {
-     interpolate,
-     spring,
-     useCurrentFrame,
-     useVideoConfig,
-   } from "remotion";
-   import { CAMERA_DISTANCE } from "./helpers/layout";
```

Comment:
Were these import statements and the one at line 290 causing the Vercel build to fail?

Reply:
I think it was `style="color: red"`. Fixed that for you!

Reply:
Perfect, thank you!